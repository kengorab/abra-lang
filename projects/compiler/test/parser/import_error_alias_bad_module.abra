import 123 as abc