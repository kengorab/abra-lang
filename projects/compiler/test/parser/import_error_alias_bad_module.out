Error at %FILE_NAME%:1:8
Unexpected token 'int', expected one of 'string', 'identifier':
  |  import 123 as abc
            ^